### DISCUSSION TITLE
RuntimeError: "LayerNormKernelImpl" not implemented for 'Half'

### DISCUSSION THREAD

**Discussion by ExecuteAI** [vladmandic/sdnext#456] [Q&A]
I stil get this runtimeerror: "LayerNormKernelImpl" not implemented for 'Half'

**Comment by vladmandic** [comment]
you just posted same in a different thread, so continue there. cross-posting is bad form.

**Reply by ExecuteAI** [reply]
OK, please delete this one then. Thanks!

**Reply by ExecuteAI** [reply]
Hi Vlad, I still have this error? RuntimeError: "LayerNormKernelImpl" not implemented for 'Half'

**Comment by ExecuteAI** [comment]
I was in launc.py and set a command, but that did not work so I deleted this again.
